File: Entities/Concrete/Customer.cs
using Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Concrete
{
    public class Customer : IEntity
    {
        public int Id { get; set; }
        public int CreatedUserId { get; set; }
        public DateTime CreatedDate { get; set; }
        public int LastUpdatedUserId { get; set; }
        public DateTime LastUpdatedDate { get; set; }
        public bool Status { get; set; }
        public bool IsDeleted { get; set; }



        public string CustomerName { get; set; }
        public string CustomerCode { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
    }
}

File: UIPreparation/UI/src/app/core/models/customer.ts
export class Customer {
    id: number = 0;
    createdUserId:number=0;
    createdDate:Date=new Date();
    lastUpdatedUserId:number=0;
    lastUpdatedDate:Date=new Date();
    status:boolean=true;
    isDeleted:boolean=false;
    customerName:string="";
    customerCode:string="";
    address:string="";
    phoneNumber:string="";
    email:string="";
}